### PR TITLE
Device tracker based on honor 3 router

### PR DESCRIPTION
**Pull request by juacas**
Component to integrate the Huawei Honor X3 and Honor Router 3 (tested on model Honor X3 Pro and Honor Router 3).

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @juacas, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#467).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/honor_x3 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "honor_x3",
+  "owner": ["@juacas"],
+  "manifest": "https://raw.githubusercontent.com/juacas/honor_x3/master/custom_components/honor_x3/manifest.json",
```
Comment (ludeeus) [inline components/honor_x3:4]: `requests` is a core requirement of Home Assistant, you should not have this in your manifest file.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: components/honor_x3 (hunk 1 above)
Comment (juacas) [reply components/honor_x3:4]: Thanks. Now it is removed from manifest.

**Review by frenck — CHANGES_REQUESTED** [review]
The filename is incorrect, please rename it from `honor_x3` to `honor_x3.json`.

**Review by frenck — CHANGES_REQUESTED** [review]
This integration has a hard pin on a generic Python dependency, used by other integrations and parts of Home Assistant.

```
  "requirements": ["bs4", "pycryptodome==3.9.9"],
```

Please loosely pin generic dependencies.

**Comment by frenck** [comment]
Not sure why a review was requested? Nothing changed...

**Comment by juacas** [comment]
I'm sorry for the annoyance. I changed the dependency version in a different requirements.txt instead the right one.

**Review by frenck — APPROVED** [review]
Thanks, @juacas 👍

**Pull request merged**